Source/UZKFileInfo.m

```diff
@@ -105,4 +107,20 @@ - (NSDate *)readDate:(tm_unz)date
     return [calendar dateFromComponents:components];
 }
 
+- (BOOL)isDirectoryWith:(unz_file_info64 *)fileInfo {
```

Comment:
This is a good idea to pass:
```mm
- (BOOL)isDirectory
{
  Return ...;
}
```

Reply:
What I actually meant was something more like this:

```ObjC
+ (BOOL)itemIsDirectory:(unz_file_info64 *)fileInfo {
    return ...;
}
```

That makes it a class method instead of an instance method. Since it's not referring to any of the instance's state, I think this would be better.